src/tools/api.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+const gamesApi = axios.create({
+  baseURL: "https://helpful-fox-leather-jacket.cyclic.app/api",
+});
+
+export const getReviews = () => {
+  return gamesApi.get("/reviews").then((res) => {
+    return res.data;
```

Comment:
Here you return `res.data`. Best to keep this consistent with the other functions and return `res.data.reviews`